### PR TITLE
User Settings Page and Ability to Cancel Subscription

### PR DESCRIPTION
**Pull request by abquansah**
The pull request encompasses:

- Styling for `UserSettingsPage`, displaying of date to cancel membership by if post not submitted, display of membership expiration date
- Cancel Modal that gives users ability to confirm if they want to cancel
- Logic upon log in that checks users `cancel_at` date and determines if user should be cancelled
- Updated `/posts` route to update user `cancel_at` date every time there is a post submission
- Added ability to cancel user subscription on front end in Stripe
- Added logic in database routes to convert Unix dates to date format in PostGreSQL via to_timestamp

### CODE REVIEW

**Comment by abquansah** [comment]
Also, I added an expiration to the JWT, so it should expire in one day

File: bloggies_frontend/src/CancelModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React, { Fragment, useState } from "react";
+import { Modal, Button } from "react-bootstrap";
+import styled from "styled-components";
+import { cancelPremiumUserMembership } from "../redux/actionCreators";
+import { useDispatch, useSelector } from "react-redux";
+import { useHistory } from "react-router";
+import { CustomReduxState } from "../custom";
+
+const ButtonWrapper = styled.div`
```
Comment (dogpineapple) [inline bloggies_frontend/src/CancelModal/index.tsx:9]: maybe we don't need this anymore xD

File: bloggies_frontend/src/RoutedPages/ComposePage/index.tsx [hunk 2]
```diff
@@ -22,7 +22,8 @@ function ComposePage() {
   }
 
   const postBlog = async (data: PostFormData) => {
-    dispatch(addPostToAPI(data));
+     dispatch(addPostToAPI(data));
+     dispatch(getUserInfoFromAPI());
```
Comment (dogpineapple) [inline bloggies_frontend/src/RoutedPages/ComposePage/index.tsx:26]: i know it wasn't part of your ticket, but would you like to add an additional check to ensure that the user's `membership_status` is `ACTIVE` when users try to go to the `ComposePage`? If `not ACTIVE`, display an error that says something like `Only premium members can contribute posts.`

lmk what you think!

**Review by stefanosu — APPROVED** [review]

**Review by dogpineapple — COMMENTED** [review]
WOOOO! It looks great!! I just left two comments as suggestions. Let me know what you think!

**Review by megmcdevitt — APPROVED** [review]
Looks great!

**Pull request merged**